src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -35,7 +37,12 @@ internal static partial class CFamilyHelper
         public const string CPP_LANGUAGE_KEY = "cpp";
         public const string C_LANGUAGE_KEY = "c";
 
-        internal static readonly string PchFilePath = Path.Combine(Path.GetTempPath(), "SonarLintForVisualStudio.PCH.preamble");
+        internal static string WorkingDirectory { get; } = Path.GetTempPath();
```

Comment:
Cosmetics, we kept repeating `Path.GetTempPath()` everywhere.

Reply:
Strictly speaking you've changed the behaviour so the temp path is stored the first time it is used and never updated.
`internal static string WorkingDirectory => Path.GetTempPath();`
...would fetch the current temp path each time it is needed.

Reply:
Yes I thought that was ok, we don't really need it to be calculated each time. It's unlikely that the temp file would change, especially not mid-VS session.

Reply:
Unlikely but not impossible. The implementation effort is the same so there doesn't seem to be a good reason to choose the less dynamic option.

BTW adding a review comment saying a change is cosmetic is a bit misleading if you've deliberately changed the behaviour!

Reply:
You're right, I didn't think of it a behavior change, sorry 🙈